### PR TITLE
Run pytest on every pr.

### PR DESCRIPTION
**Pull request by azizova-leyla**
Fixes https://github.com/civiform/civiform/issues/2888

### CODE REVIEW

File: .github/workflows/run_pytest.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: PyTest
+
+on:
+  # code review question: this commits directly to PR branches.
+  # that seems like what we're looking for, but we could also format only
+  # committed code.
```
Comment (nbeloglazov) [inline .github/workflows/run_pytest.yaml:6]: I don't understand that. The action is just running test. Why are are talking about "commiting" and "formatting"?

File: cloud/shared/variable_definitions.json [hunk 2]
```diff
@@ -41,6 +41,12 @@
     "tfvar": true,
     "type": "string"
   },
+  "ADFS_ADMIN_GROUP": {
```
Comment (nbeloglazov) [inline cloud/shared/variable_definitions.json:44]: Was this variable lost in some previous PR?

File: .github/workflows/run_pytest.yaml (hunk 1 above)
Comment (azizova-leyla) [reply .github/workflows/run_pytest.yaml:6]: Copied it from somewhere else. Not applicable here.

File: cloud/shared/variable_definitions.json (hunk 2 above)
Comment (azizova-leyla) [reply cloud/shared/variable_definitions.json:44]: it was in both aws and azure configs, and currently that's not allowed. I'm moving it to shared config.
cloud/shared/validate_variable_definitions_test.py is checking for that and it wasn't running before this change.

**Review by nbeloglazov — APPROVED** [review]

**Pull request merged**